### PR TITLE
Make references to ISAs more generic.

### PR DESCRIPTION
**Pull request by gvnn3**
This is a first cut at how I'd like to expand the exercises to properly cover morello as well as RISC-V.

### CODE REVIEW

File: src/introduction/README.md [hunk 1]
```diff
@@ -2,10 +2,12 @@
 
 This set of exercises and adversarial missions is intended to:
 
-- Build a baseline skillset with RISC-V and CHERI-RISC-V, as well as awareness
+- Build a baseline skill set with unmodified instruction set
+  architectures (ISAs) such as ARMv8 and RISC-V as well as the same
```
Comment (jrtc27) [inline src/introduction/README.md:6]:
```suggestion
  architectures (ISAs) such as ARMv8 and RISC-V and the same
```
given we use "as well as" again later in the sentence.

File: src/introduction/README.md [hunk 2]
```diff
@@ -16,40 +18,40 @@ and software stacks.
...
-and ask you to build and perform minor modifications to simple
-RISC-V and CHERI-RISC-V C/C++ programs. These exercises
-facilitate building skills such as compiling, executing,
-and debugging RISC-V and CHERI-RISC-V programs, as well as to build basic
-understanding of CHERI C/C++ properties. We highlight some key edge
-cases in CHERI, including the effects of bounds imprecision, subobject
-bounds, weaker temporal safety, and C type confusion.
+and ask you to build and perform minor modifications to simple C/C++
+programs. These exercises facilitate building skills such as
+compiling, executing, and debugging programs with both CHERI enabled
```
Comment (jrtc27) [inline src/introduction/README.md:23]: Maybe "with and without CHERI extensions"?

File: src/introduction/README.md [hunk 3]
```diff
@@ -16,40 +18,40 @@ and software stacks.
...
+and default ISAs, as well as to build basic understanding of CHERI
+C/C++ properties. We highlight some key edge cases in CHERI, including
+the effects of bounds imprecision, subobject bounds, weaker temporal
+safety, and C type confusion.
 
 These exercises take for granted a strong existing understanding of:
 - The C/C++ languages
 - UNIX program compilation, execution, and debugging
-- RISC ISAs and binary structures/reverse engineering (e.g., on MIPS or ARMv8)
+- ARMv8 and/or RISC ISAs as well as binary structures/reverse engineering
```
Comment (jrtc27) [inline src/introduction/README.md:32]: The original was fine.

File: src/introduction/README.md [hunk 4]
```diff
@@ -16,40 +18,40 @@ and software stacks.
...
+overflows combined with ROP). For those familiar with conventional
+low-level attack techniques, this will also act as a warm-up exercise
+on the baseline architecture and expand experience with reverse
 engineering and exploitation.
 
-The CHERI-RISC-V team has confirmed exploitability for the RISC-V binary
-in advance.  We strongly recommend exploiting the RISC-V version of the code
-first, as a starting point for understanding potential CHERI-RISC-V
-exploitability.
+The CHERI team has confirmed exploitability for the non CHERI enabled
```
Comment (jrtc27) [inline src/introduction/README.md:54]: baseline?

File: src/introduction/README.md [hunk 5]
```diff
@@ -16,40 +18,40 @@ and software stacks.
...
+- ARMv8 and/or RISC ISAs as well as binary structures/reverse engineering
 
 ## Focused adversarial missions
 **Focused adversarial missions** are intended to take
-1-3 days, and ask you to exploit, first on RISC-V, and
-then on CHERI-RISC-V, documented vulnerabilities in simple "potted"
-C/C++-language programs provided by the CHERI-RISC-V team. These missions
-engage you more specifically in RISC-V exploitation, and CHERI's
+1-3 days, and ask you to exploit, first on an unmodified ISA, and
+then on a CHERI enabled ISA, documented vulnerabilities in simple "potted"
```
Comment (jrtc27) [inline src/introduction/README.md:37]:
```suggestion
then on a CHERI-enabled ISA, documented vulnerabilities in simple "potted"
```
(repeated)

File: src/introduction/README.md [hunk 6]
```diff
@@ -16,40 +18,40 @@ and software stacks.
...
 engineering and exploitation.
 
-The CHERI-RISC-V team has confirmed exploitability for the RISC-V binary
-in advance.  We strongly recommend exploiting the RISC-V version of the code
-first, as a starting point for understanding potential CHERI-RISC-V
-exploitability.
+The CHERI team has confirmed exploitability for the non CHERI enabled
+binaries in advance.  We strongly recommend exploiting the non CHERI
+enabled version of the code first, as a starting point for
+understanding potential CHERI-RISC-V exploitability.
```
Comment (jrtc27) [inline src/introduction/README.md:57]:
```suggestion
understanding potential exploitability of the CHERI version.
```

**Comment by jrtc27** [comment]
Generally seems like the right approach, though you may find things get a bit more awkward in some of the later sections.

File: src/introduction/README.md (hunk 2 above)
Comment (gvnn3) [reply src/introduction/README.md:23]: Reworked in the file.

File: src/introduction/README.md (hunk 4 above)
Comment (gvnn3) [reply src/introduction/README.md:54]: Yeah, it's awkward but I don't think baseline works either.  We can revisit in a bit.

File: src/introduction/README.md (hunk 3 above)
Comment (gvnn3) [reply src/introduction/README.md:32]: Well, I think we need to call out both but not in a parenthetical.

Comment (jrtc27) [reply src/introduction/README.md:32]: ARMv8 and RISC-V are both specific instances of RISC ISAs

**Pull request merged**